src/redux/skybrowser/skybrowserSlice.ts

```diff
@@ -0,0 +1,119 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export interface SkyBrowserImage {
```

Comment:
These data types (`SkybrowserImage`, `SkyBrowserBrowser` and `SkybrowserState`) would benefit from descriptions of what the different parameters are.  Now one has to know how the skybrowser works to understand that 

Also specify if angles are radians/degrees